source/user-manual/manager/index.rst

```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Find out more about Wazuh server administration and its configurations in this section of our documentation. 
```

Comment:
I can still see the old meta description under the title of this section (lines 8/10) and it needs to be removed.
If you've already fixed it later on, dismiss this comment.